cleanup-passes/travis.ts

```diff
@@ -75,7 +64,8 @@ async function cleanupTravisConfig(element: ElementRepo): Promise<void> {
 
   // use ubuntu trusty
   travis.dist = 'trusty';
-  travis.sudo = 'required';
+  // use docker
+  travis.sudo = 'false';
```

Comment:
This can just be left off then, yeah?

Reply:
No it has to be explicitly set to `'false'` to use docker instances.